designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/internal/DateTimeUtils.kt

```diff
@@ -7,13 +7,22 @@ import java.text.ParseException
 import java.text.SimpleDateFormat
 import java.util.Calendar
 
+@SuppressWarnings("deprecation")
+@Deprecated("This class is internal and should not be used replace with the new implementation")
 internal fun dateIsInRange(date: Long, allowedDates: SelectableDates, format: String = "ddMMyyyy"): Boolean {
```

Comment:
if the class is for internal use, why deprecate it and not changing it?. It shouldn't provoke breaking changes, right?

Reply:
I have deprecated the one that is used by the old component although it won't create breaking changes it is no longer needed and I will remove it when we remove the deprecated component